extended/src/main/java/apoc/cypher/CypherExtended.java

```diff
@@ -369,19 +369,20 @@ public Stream<MapResult> mapParallel2(@Name("fragment") String fragment, @Name("
         final String statement = withParamsAndIterator(fragment, params.keySet(), "_");
         tx.execute("EXPLAIN " + statement).close();
         BlockingQueue<RowResult> queue = new ArrayBlockingQueue<>(100000);
-        Stream<List<Object>> parallelPartitions = Util.partitionSubList(data, (int)(partitions <= 0 ? PARTITIONS : partitions), null);
+        List<List<Object>> parallelPartitions = Util.partitionSubList(data, (int)(partitions <= 0 ? PARTITIONS : partitions), null)
```

Comment:
Secondo me questa non è una risoluzione perché vorrebbe dire ricaricare in memoria i dati...
Prova a scrivere un piccolo documento da condividere con Louise e Nacho per capire se investigare lato db è un'alternativa.
Digli che te l'ho chiesto io.